.ci/azure/windows.yml

```diff
@@ -104,6 +104,7 @@ jobs:
       $(PYTHON_EXE) -m pip install -r $(OPENVINO_REPO_DIR)\src\bindings\python\requirements.txt
       $(PYTHON_EXE) -m pip install -r $(REPO_DIR)\modules\custom_operations\tests\requirements.txt
       $(PYTHON_EXE) -m pip install $(OPENVINO_REPO_DIR)\tools\mo
+      $(PYTHON_EXE) -m pip install $(REPO_DIR)\modules\custom_operations\.[all]
```

Comment:
we need to call setupvars.bat to provide OpenVINO_DIR for ov-tokenizers building